test/core/utils/finalize-result.js

```diff
@@ -1,46 +1,16 @@
 describe('axe.utils.finalizeRuleResult', function() {
 	'use strict';
 
-	var origAggregate;
-	beforeEach(function() {
-		origAggregate = axe.utils.aggregateNodeResults;
-	});
-
-	afterEach(function() {
-		axe.utils.aggregateNodeResults = origAggregate;
-	});
-
 	it('should be a function', function() {
 		assert.isFunction(axe.utils.finalizeRuleResult);
 	});
 
 	it('returns the first param object', function() {
-		axe.utils.aggregateNodeResults = function() {};
-
-		var goingIn = {};
+		var goingIn = {
+			nodes: []
+		};
 		var comingOut = axe.utils.finalizeRuleResult(goingIn);
 
 		assert.equal(goingIn, comingOut);
 	});
-
-	it('passes the nodes property to utils.aggregateNodeResults', function() {
```

Comment:
Agreed these are bad tests, but I'd feel much better if we replaced it instead of just taking it out.